github/activity_events.go

```diff
@@ -30,7 +30,7 @@ func (e Event) String() string {
 
 // Payload returns the parsed event payload. For recognized event types,
 // a value of the corresponding struct type will be returned.
-func (e *Event) Payload() (payload interface{}) {
+func (e *Event) Payload() (payload interface{}, err error) {
```

Comment:
Given this is a breaking API change anyway, I think we should also rename the method to be a better fit for what it does.

`Payload` was okay when it returned a single value and never failed.

Now, I think a better name would be:

```Go
// ParsePayload parses the event payload. For recognized event types,
// a value of the corresponding struct type will be returned.
func (e *Event) ParsePayload() (payload interface{}, err error) {
```

It'll be similar to `ParseWebHook`.